.eslintrc.js

```diff
@@ -42,6 +43,8 @@ module.exports = {
         'import/extensions': 'off',
         'no-use-before-define': 'off',
         'no-shadow': 'off',
+        'no-unused-expressions': 'off',
```

Comment:
Hello, 

I'm disabling these rules only for a Typescript context, because i am enabling the more suitable Typescript-eslint rules with rules like this : 

`@typescript-eslint/no-unused-expressions': ['error']`